src/backend/core/models.py

```diff
@@ -452,6 +454,34 @@ def get_abilities(self, user):
         }
 
 
+class Invitation(BaseModel):
+    """User invitation to teams."""
+
+    email = models.EmailField(_("email address"), null=False, blank=False)
+    team = models.ForeignKey(
+        Team,
+        on_delete=models.CASCADE,
+        related_name="invitations",
+    )
+    role = models.CharField(
+        max_length=20, choices=RoleChoices.choices, default=RoleChoices.MEMBER
+    )
+    issuer = models.ForeignKey(
+        User, on_delete=models.SET_NULL, related_name="invitations", null=True
+    )
+
+    def __str__(self):
+        return f"{self.email} invited to {self.team}"
+
+    @property
+    def is_pending(self):
+        """Calculate if invitation is still valid or has expired."""
+        tz = pytz.timezone(settings.TIME_ZONE)
+        return datetime.utcnow(tz=tz) < (
```

Comment:
I had to grab the project timezone in the expected format. If there is a cleaner way to do it, feel free to suggest it.

Reply:
If it's still a good practice, you can use:
```django
from django.utils import timezone
now = timezone.now()
```